localstack/services/es/es_api.py

```diff
@@ -190,13 +191,12 @@ def start_elasticsearch_instance(version):
 
     # install ES version
     install_version = get_install_version_for_api_version(version)
-    install.install_elasticsearch(install_version)
 
-    t1 = es_starter.start_elasticsearch(asynchronous=True, version=install_version)
-    # sleep some time to give Elasticsearch enough time to come up
-    time.sleep(8)
-    # ensure that all infra components are up and running
+    t1 = es_starter.start_elasticsearch(asynchronous=False, version=install_version)
```

Comment:
Not 100% sure, but I think if we pass `asynchronous=False` here, then the process will be started synchronously (i.e., the call to `start_elasticsearch(...)` won't return until the Elasticsearch process itself terminates, whenever the ES domain is stopped/deleted). 

So, I believe that lines 196-200 won't get executed (and hence can be removed), is that right?